src/com/lftechnology/java/training/srijan/Test.java

```diff
@@ -0,0 +1,29 @@
+package com.lftechnology.java.training.srijan;
+import java.util.Scanner;
+
+public class Test {
+	
+	public static void main(String[] args){
+		
+		Scanner input = null;
+		
+		try{
+			input = new Scanner(System.in);
+			System.out.println("Enter your Name with Salutation");
+			String nameWithSalutation = input.nextLine();
+			System.out.printf("%n%s%n", nameWithSalutation);
+			
+			String[] text = nameWithSalutation.split(" ");
+			System.out.println("Salutation:" + text[0]);
```

Comment:
before you try text[0] you should check of null, otherwise there might be chance of NPE(Null Pointer Exception)